packages/suite/src/components/suite/FormattedCryptoAmount/index.tsx

```diff
@@ -18,7 +20,7 @@ const FormattedCryptoAmount = ({ value, symbol, disableHiddenPlaceholder, classN
     const content = (
         <>
             <Value>{value}</Value>
-            {symbol && <Symbol>{` ${symbol.toUpperCase()}`}</Symbol>}
```

Comment:
are we using `&nbsp;` now? i thought i saw your discussion with @keraf where you convinced him that it's better to use `{' '}` instead?
anyway, i already did a little change in this component here so we may have a conflict 
https://github.com/trezor/trezor-suite/pull/2626/commits/2e5a65eadd15ca55d7f4385861af00a70d71544e

Reply:
`&nbsp;` has its use, it is literally non-breaking space. It is good to know that it is not 1:1 replacement for a space (if you would use nbsp everywhere instead of regular space, you could end up with funky line breaking behaviour).

I used it here, cause it seems that nothing else really works if the parent of `FormattedCryptoAmount` is flex container (and I need it in token list to properly truncate crypto amounts). I assume that there won't be a case where we would want to have `Value` and `Symbol` break over 2 lines so nbsp seems okay here.